test/art_vandelay_test.rb

```diff
@@ -426,13 +426,23 @@ class Import < ArtVandelayTest
         ],
         result.rows_accepted
       )
-      assert_equal(
-        [
-          row: ["invalid@example.com", nil],
-          errors: {password: ["can't be blank"]}
-        ],
-        result.rows_rejected
-      )
+      if Rails.version >= "7.1"
+        assert_equal(
+          [
+            row: ["invalid@example.com", nil],
+            errors: {password: ["can’t be blank"]}
+          ],
+          result.rows_rejected
+        )
+      else
+        assert_equal(
+          [
+            row: ["invalid@example.com", nil],
+            errors: {password: ["can't be blank"]}
+          ],
+          result.rows_rejected
+        )
+      end
```

Comment:
This specific issue actually came up in a recent discussion! Although this approach works, we have an opportunity to further improve it by using [built in translations for Active Record](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models).

```suggestion
      assert_equal(
        [
          row: ["invalid@example.com", nil],
          errors: {password: [I18n.t("errors.messages.blank")]}
        ],
        result.rows_rejected
      )
```

Here's[ the full list of validation translations](https://guides.rubyonrails.org/i18n.html#error-message-interpolation) for context.

Reply:
Oh nice! That's much cleaner. I'll make that change.